DashAI/front/src/components/Upload.jsx

```diff
@@ -1,7 +1,8 @@
 import React, { useState, useRef } from "react";
 import { useNavigate } from "react-router-dom";
 import PropTypes from "prop-types";
-import { P, Title, StyledButton, Loading } from "../styles/globalComponents";
+import { P, Title, Loading } from "../styles/globalComponents";
```

Comment:
Why not use directly material components instead?

Reply:
`Loading` can be replaced by [progress](https://mui.com/material-ui/react-progress/).

Reply:
thank you.
I think it would be simpler to incorporate those changes in PR #67, because it includes a refactoring of `Upload.jsx`.

Reply:
Nice!
Closing conversation.